config/initializers/01_tag_presentation.rb

```diff
@@ -0,0 +1,35 @@
+module TagPresentation
```

Comment:
Is the name of initializer prefixed by a `01` a new procedure? Seems like a different naming scheme than we've been using in the past.

Reply:
Yeah its so it gets loaded first I assume, once again just copying the way topology is doing it.